test/quantization/core/test_quantized_op.py

```diff
@@ -253,7 +253,29 @@ def test_qrelu(self, X):
                 }
             }
         ]
-        self._test_activation_function(X, 'relu', relu_test_configs)
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
+        for device in devices:
+            # Only test the non-in-place version relu quantized cuda,
+            # will remove this when creating in-place version relu quantized cuda.
+            if device == 'cuda':
+                relu_test_configs = [
+                    {
+                        'quantized_fn': [
+                            torch.relu,
+                            torch.nn.functional.relu,
+                        ],
+                        'reference_fn': torch.nn.functional.relu
+                    },
+                ]
```

Comment:
nit: FYI, this is not a right way to "temporarily" override the configurations. The problem is that `relu_test_configs` is defined for the whole test. However, within the loop you are overriding it, meaning on all next iterations it will be the "overriden" version.

It's OK for this PR, as the next one removes these lines.

Reply:
OK, will avoid this next time.